IdentityCore/src/webview/embeddedWebview/challangeHandlers/MSIDPKeyAuthHandler.m

```diff
@@ -72,6 +73,23 @@ + (BOOL)handleChallenge:(NSString *)challengeUrl
     NSMutableURLRequest *responseReq = [[NSMutableURLRequest alloc] initWithURL:responseUrlComp.URL];
     [responseReq setValue:kMSIDPKeyAuthHeaderVersion forHTTPHeaderField:kMSIDPKeyAuthHeader];
     [responseReq setValue:authHeader forHTTPHeaderField:MSID_OAUTH2_AUTHORIZATION];
+
+    // Adding refreshTokenCredential (PRT) header to the challenge response. Header is available in customheaders dict
+    BOOL refreshTokenIsValid = NO;
```

Comment:
nit: can get rid of this BOOL as it is only used to check if log statement should be done. Can include the log statement as part of else on line 86